src/components/render-page/window-post-message-communicator/editor-to-renderer-communicator.ts

```diff
@@ -15,7 +16,7 @@ export class EditorToRendererCommunicator extends WindowPostMessageCommunicator<
   EditorToRendererMessageType,
   CommunicationMessages
 > {
-  protected generateLogIdentifier(): string {
-    return 'E=>R'
+  protected createLogger(): Logger {
+    return new Logger('WPMC[E>R]')
```

Comment:
We have plenty of space. Maybe use a bit more explanatory terms instead of these abbreviations. We know what that means, but other people that might want to debug something later on, don't know what that means.